### PR TITLE
Added MeshOps v1 Support

### LINKED ISSUE #33: MeshOps v1 Support
#### Current Behavior
Each Meshery Adapter should support MeshOps v1 style operations and a listing of its operations.

The adapter for Cilium returns this today:

``[{"adapter_location":"localhost:10012","name":"CILIUM_SERVICE_MESH","version":"edge","git_commit_sha":"none","ops":null},```

#### Expected Behavior
All available operations should be listed in `ops`.

---
#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by MUzairS15**
Signed-off-by: MUzairS15 <muzair.shaikh810@gmail.com>

**Description**

This PR fixes #33 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by MUzairS15** [comment]
// @leecalcote

File: internal/config/config.go [hunk 1]
```diff
@@ -81,6 +81,11 @@ func New(provider string) (h config.Handler, err error) {
 		return nil, err
 	}
 
+	// Setup Operations Config
+	if err := h.SetObject(adapter.OperationsKey, Operations); err != nil {
+		return nil, err
```
Comment (leecalcote) [inline internal/config/config.go:86]: Should this be using MeshKit style errors? https://docs.meshery.io/project/contributing/contributing-error

Comment (MUzairS15) [reply internal/config/config.go:86]: I don't think so, the error is not encountered due to any network error. (As far as I understand)

Comment (leecalcote) [reply internal/config/config.go:86]: Network or not…. Doesn’t matter.  @Revolyssup  shine some light this, please.

**Review by theBeginner86 — APPROVED** [review]
Thanks, @MUzairS15. With these changes, the backend seems sorted. 👍

Though the operations get loaded when we are on the `/management` route. But there seems a typo in the cilium's API because of which the navigator is not able to load the Cilium's lifecycle operations.
Could you please have a look at the UI as well? 

For reference:

https://user-images.githubusercontent.com/73700530/159974037-9c7b8187-f2d9-434b-83d4-28ecff222f6b.mov

**Comment by MUzairS15** [comment]
> Thanks, @MUzairS15. With these changes, the backend seems sorted. 👍
> 
> Though the operations get loaded when we are on the `/management` route. But there seems a typo in the cilium's API because of which the navigator is not able to load the Cilium's lifecycle operations. Could you please have a look at the UI as well?
> 
> For reference:
> 
>  Screen.Recording.2022-03-24.at.10.47.03.PM.mov

Yes will look into it today.

File: internal/config/config.go (hunk 1 above)
Comment (MUzairS15) [reply internal/config/config.go:86]: can someone pls help??

**Comment by MUzairS15** [comment]
@theBeginner86 can you pls check for nginx sm and open sm.
I experiencing same issue with nginx sm. Can you pls confirm?

**Comment by leecalcote** [comment]
I believe that the issue lies within string matching of the service mesh and service mesh adapter names happening in Meshery UI.

https://user-images.githubusercontent.com/7570704/160146029-e215cc3f-7a9f-49d9-977d-b5464a53325e.mov

**Comment by MUzairS15** [comment]
Yes @leecalcote you are correct I have raised the PR for the fix

**Comment by leecalcote** [comment]
> Yes @leecalcote you are correct I have raised the PR for the fix

@MUzairS15 which PR? This one?

File: internal/config/config.go [hunk 2]
```diff
@@ -83,7 +83,7 @@ func New(provider string) (h config.Handler, err error) {
 
 	// Setup Operations Config
 	if err := h.SetObject(adapter.OperationsKey, Operations); err != nil {
-		return nil, err
+		return nil, adapter.ErrClientConfig(err)
```
Comment (MUzairS15) [inline internal/config/config.go:86]: @leecalcote Is it ok now?

Comment (leecalcote) [reply internal/config/config.go:86]: @MUzairS15 thanks for this. Yes, closer. There should be definition of long description, probable cause, and suggested remediation, too.

**Review by leecalcote — APPROVED** [review]
Thanks, @MUzairS15

File: internal/config/config.go (hunk 2 above)
Comment (MUzairS15) [reply internal/config/config.go:86]: > @MUzairS15 thanks for this. Yes, closer. There should be definition of long description, probable cause, and suggested remediation, too.

BTW, @leecalcote what abt this?

**Pull request merged**